### PR TITLE
Convert percent slope band to int16 and scale it.

### PR DESCRIPTION
**Pull request by raydittmeier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## ESPA Surface Water Extent - Release Notes
-Release Date: November 2017
+Release Date: February 2018
```
Comment (gschmidt-usgs) [inline README.md]: Release for ESPA is in March.

File: cfmask-based-water-detection/README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 
-## CFmask Based Water Detection Version 1.1.0 - Release Notes
+## CFmask Based Water Detection Version 1.1.1 - Release Notes
 
-Release Date: January 2017
+Release Date: February 2018
```
Comment (gschmidt-usgs) [inline cfmask-based-water-detection/README.md]: March 2018 for ESPA

File: dswe/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
-## Dynamic Surface Water Extent Version 2.2.0 - Release Notes
+## Dynamic Surface Water Extent Version 2.3.0 - Release Notes
 
-Release Date: November 2017
+Release Date: February 2018
```
Comment (gschmidt-usgs) [inline dswe/README.md]: March 2018

File: dswe/algorithm-description.md [hunk 4]
```diff
@@ -65,6 +66,12 @@ Algorithm Description - Overview:
     example, a pixel masked for cloud shadow, percent slope, and hillshade
     will have a value of 25 (2^0 + 2^3 + 2^4 = 1 + 8 + 16 = 25).
 
+    The 3 optional intermediate bands are:
+
+    diagnostic band - shows the results of the initial tests before recoding
+    percent slope band - built from DEM; used for percent slope filtering
+    hillshade band - built from DEM; used for hilshade filtering
```
Comment (gschmidt-usgs) [inline dswe/algorithm-description.md]: hilshade - typo

File: dswe/src/dswe.c [hunk 5]
```diff
@@ -985,10 +986,23 @@ main (int argc, char *argv[])
 
     if (include_ps_flag)
     {
+        /* Convert to a scaled 16 bit integer value */
+        for (index = 0; index < pixel_count; index++)
+        {
+            percent_slope = (band_ps[index] * PERCENT_SLOPE_MULT_FACTOR) + 0.5;
+
+            /* If the scaled value is outside the range, pull it back */
+            if (percent_slope > GDAL_INT16_MAX)
```
Comment (gschmidt-usgs) [inline dswe/src/dswe.c]: This appears to assume percent slope is never negative.  It is possible for a slope to be negative, so may need to check against the minimum INT16 value.

**Comment by gschmidt-usgs** [comment]
Peer Review complete

File: README.md (hunk 1 above)
Comment (raydittmeier) [reply README.md]: updated date

File: cfmask-based-water-detection/README.md (hunk 2 above)
Comment (raydittmeier) [reply cfmask-based-water-detection/README.md]: updated date

File: dswe/README.md (hunk 3 above)
Comment (raydittmeier) [reply dswe/README.md]: updated date

File: dswe/algorithm-description.md (hunk 4 above)
Comment (raydittmeier) [reply dswe/algorithm-description.md]: fixed typo

File: dswe/src/dswe.c (hunk 5 above)
Comment (raydittmeier) [reply dswe/src/dswe.c]: Both slope algorithms compute the combined x and y slope as sqrt(x^2 + y^2), so even if the x and/or y slope is negative, the combined slope shouldn't be negative.

**Pull request merged**